app/services/reports/calibration_report.rb

```
@@ -0,0 +1,105 @@
+        # Delegate the valid score range to the Questionnaire (Information Expert).
+        # The fallback to class-level defaults lives in Questionnaire#score_range,
+        # not here, so calibration code does not carry knowledge of score bounds.
+        range = item.questionnaire&.score_range || (0..5)
+        hash[item.id] = range.each_with_object({}) { |score, b| b[score.to_s] = 0 }
+      end
+    end
+
+    def per_item_summary(item)
+      instructor_answer = @instructor_response.answer_for(item)
+      {
+        item_id:                item.id,
```

RuboCop (1.86.1)

[convention] Align the keys of a hash literal if they span more than one line.

(Layout/HashAlignment)